tests/common.go

```diff
@@ -4397,6 +4399,61 @@ func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
 	}
 }
 
+func collectAndCopyLogsForTest(testCaseName string, podLabel map[string]string, namespaceFunc func() (string, error), logLabel string) {
```

Comment:
Why do you want to accept `namespaceFunc` simply ask for the `namespace` string

Reply:
In case of a failure of the runCmd function, it returns an error message that includes the command it ran. I am assuming, pwd might be more useful for de-bugging. This convention is taken from the legacy code.

Reply:
Done.